### PR TITLE
add shadow to bar view

### PR DESCRIPTION
**Pull request by rajkothari634**
added shadow to bar view

### CODE REVIEW

**Comment by sidhuparas** [comment]
Kindly remove unused imports as suggested in PR Quality Review and solve the conflicts.

**Review by samagragupta — COMMENTED** [review]
Resolve conflicts

**Comment by ShivangiSingh17** [comment]
@rajkothari634 solve the conflicting files and squash the commits once you are done resolving the errors.

**Comment by krharsh17** [comment]
@rajkothari634 what's the status on this?

**Comment by rajkothari634** [comment]
Improving the effect of shadow

File: library/src/main/java/in/krharsh17/barview/BarGroup.java [hunk 1]
```diff
@@ -150,10 +150,13 @@ void setupLabel() {
...
     void setupInitial() {
-        initialParams = new LayoutParams(dp(12), dp(BAR_HEIGHT));
+        initialParams = new LayoutParams(dp(12) + 10, dp(BAR_HEIGHT) + 10);
         initialParams.rightMargin = dp(12);
         initial.setLayoutParams(initialParams);
-        initial.setBackgroundColor(Color.parseColor(color));
+        Drawable[] layers = {getRoundRect(color.substring(1),0),  getRoundRect(color.substring(1),1), getRoundRect(color.substring(1),2), getRoundRect(color.substring(1),3), getRoundRect(color.substring(1),4)};
+        LayerDrawable splash_test = new LayerDrawable(layers);
+        initial.setBackground(splash_test);
+        //initial.setBackgroundColor(Color.parseColor(color));
```
Comment (ShivangiSingh17) [inline library/src/main/java/in/krharsh17/barview/BarGroup.java]: Are you trying to reflect it over the whole page activity?

File: library/src/main/java/in/krharsh17/barview/BarGroup.java [hunk 2]
```diff
@@ -260,6 +263,7 @@ public Drawable getRoundRect(String color,int layerNumber) {
         }, null, null);
 
         ShapeDrawable shapeDrawable = new ShapeDrawable(rectShape);
+
```
Comment (ShivangiSingh17) [inline library/src/main/java/in/krharsh17/barview/BarGroup.java]: Avoid adding unnecessary spaces.

**Review by ShivangiSingh17 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**